src/mmw/js/src/modeling/templates/controls/conservationPractice.html

```diff
@@ -7,14 +7,14 @@
     <div id="conserve-tools" class="dropdown-menu draw-tools menu-left" role="menu"> <!-- Dropdown -->
         <div class="pad-1"> <!-- Dropdown Content -->
             <ul class="row">
-                {{ utils.thumb('Rain Garden', 'rain_garden') }}
-                {{ utils.thumb('Veg Infil Basin', 'veg_infil_basin') }}
-                {{ utils.thumb('Porous Paving', 'porous_paving') }}
+                {{ utils.thumb(getHumanReadableLabel('rain_garden'), 'rain_garden') }}
```

Comment:
This is a purely nice-to-have though, I don't think we need to spend too much time on it if it isn't very straightforward.